specification/fist/IotFirmwareDefense.Management/tspconfig.yaml

```diff
@@ -16,6 +16,11 @@ options:
     flavor: "azure"
     generate-test: true
     generate-sample: true
+  "@azure-tools/typespec-java":
+    package-dir: "azure-resourcemanager-iotfirmwaredefense"
+    namespace: "com.azure.resourcemanager.iotfirmwaredefense"
+    flavor: "azure"
+    service-name: "Iot Firmware Defense"
```

Comment:
IoT

link to namespace PR?

Reply:
Link to namespace review : https://github.com/Azure/azure-sdk-pr/issues/1126

Reply:
IoT

Reply:
service-name changed to  "IoT Firmware Defense".